.github/workflows/composite/docker-builder-agw/action.yml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build Docker Image For AGW
+description: Build a docker image and push it to a registry
+
+inputs:
+  REGISTRY_USERNAME:
+    description: Registry login username
+    required: true
+  REGISTRY_PASSWORD:
+    description: Registry login password
+    required: true
+  REGISTRY:
+    description: Docker registry
+    required: true
+  IMAGE:
+    description: Docker image stream name
+    required: true
+  TAG:
+    description: Docker image stream tag
+    default: latest
+  DOCKERFILE:
+    description: Docker file
+    required: false
+    default: Dockerfile
+  DOCKERCONTEXT:
+    description: Docker context
+    default: .
+  PUSH:
+    description: Push to registry?
+    # See unresolved bug https://github.com/actions/runner/issues/1483
```

Comment:
The same issue was relevant in https://github.com/magma/magma/pull/14226/files.
I think it still makes sense to use `type: boolean` here.